Updated the "client cache minimum retention period" (#1826)
MicrosoftDocs/SCCMdocs: sccm/core/plan-design/changes/whats-new-in-version-1906.md

### Configure client cache minimum retention period

<!--4485509-->

> You can now specify the minimum time for the Configuration Manager client to keep cached content. This client setting defines the minimum amount of time Configuration Manager agent should wait before it can remove a content from the cache in case more space is needed. In the **Client cache settings** group of client settings, configure the following setting: **Minimum duration before cached content can be removed (minutes)**.

Reviewer: @mestew will need to signoff, but here's my suggested rewording:
```suggestion
You can now specify the minimum time for the Configuration Manager client to keep cached content. When the client needs more space in its cache, this client setting defines the minimum amount of time the client waits before it removes content from the cache. In the **Client cache settings** group of client settings, configure the following setting: **Minimum duration before cached content can be removed (minutes)**.
```

Once finalized, we also need to update the same string in the [Client settings](https://docs.microsoft.com/sccm/core/clients/deploy/about-client-settings#minimum-duration-before-cached-content-can-be-removed-minutes) article. @kerimhanif do you want to change that one as well? If not, I can update once this PR is complete.

Author: I like your suggestion!
